PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/appservice/Platform.java

```diff
@@ -0,0 +1,130 @@
+package com.microsoft.azure.toolkit.appservice;
+
+import com.google.common.collect.ImmutableList;
+import com.microsoft.azure.management.appservice.OperatingSystem;
+import lombok.Getter;
+
+import java.util.List;
+
+public interface Platform {
+
+    OperatingSystem getOs();
+
+    String getStackOrWebContainer();
+
+    String getStackVersionOrJavaVersion();
+
+    List<Platform> platforms = ImmutableList.copyOf(new Platform[]{
```

Comment:
Shall we move the value to a constant class?

Reply:
No, I don't think so. if you want to find platforms, go to `Platform`.